enclave_mgmt/compile_models.py

```diff
@@ -383,18 +409,55 @@ def collect_oneroster_dfs(bucket, key):
     }
 
 
-def compile_models(cli_bucket, cli_key, oneroster_bucket, oneroster_key):
+def collect_quiz_dfs(bucket, key):
+    key_questions = key + "/contents/quiz_questions.csv"
+    key_question_contents = key + "/contents/quiz_question_contents.csv"
+    key_multichoice_answers = key + "/contents/quiz_multichoice_answers.csv"
```

Comment:
Minor-ish (but super important we're consistent :joy: ): We we want `/contents/` or `/content/`? We were discussing the latter [here](https://github.com/openstax/k12/issues/307#issuecomment-1387497365), but maybe this is better. I just want to be sure we explicitly decide on the convention either way :man_shrugging: .

Reply:
Hmm, yea good point. They kind of imply different things - for that reason I think I'd rather go with content. I'll make that tweak to `/content/`